app/src/main/java/com/github/ybecker/epforuml/HomeFragment.kt

```diff
@@ -51,7 +54,7 @@ class HomeFragment(val mainActivity: MainActivity) : Fragment() {
         // remove when using mockDB
         //getQuestionsQuery()
 
-        futureCourseList.thenAccept {
+        futureCourseList.thenAccept { it ->
```

Comment:
I thinks that here you should either use a lambda function with an explicit name other than 'it', or not use any lambda and use the Kotlin "it" instruction.